modules/meshrenderinggl/glsl/oit/sort.glsl

```diff
@@ -46,7 +46,7 @@ void fillFragmentArray(uint idx, out int numFrag);
 // Fill local memory array with lowest values
 void fillFragmentArraySortedUntilDepth(uint pixelIdx, float maxDepth, out int numFrag);
 // Selection sort, allowing to get the first elements of a larger series
-vec4 selectionSortNext(uint headPtr, float lastDepth);
+vec4 selectionSortNext(uint headPtr, float lastDepth, inout uint lastPtr);
```

Comment:
Should also add documentation about the new variable.

Reply:
Should be clearer now.